### ISSUE TITLE
type checking for lines()

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#2394]
and other built-in functions?

this should give syntax error,

```abap
DATA int TYPE i.
WRITE lines( int ).
```
